experiments/henry-parallelization-runner.sh

```diff
@@ -0,0 +1,71 @@
+#source ./config.sh
```

Comment:
Please change these file names to be something more descriptive. henry-X.sh does not inform me much about how the script differs from the existing ones. Perhaps something like parallelization-w-arguments.sh is more appropriate.